README.md

```diff
@@ -40,31 +40,42 @@ east student silly already breeze enact seat trade few way online skin grass hum
   <summary>How to generate a root private key</summary>
 
 ```
-$ cardano-address recovery-phrase generate --size 15 > recovery-phrase.prv
-$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley
-xprv1fzu4e8cecxshgzzxzh7557sd8tffqreeq2je7fgsm7f02mq849vdupw7qwgxc3qawyqev0l8ew0f4fkp8hvr8mskz4hz6e6ejzjlevcskcl6lqpr07u7552fsfgteztuclse7luh4cp493zdhkrjdss0250cdw8n
+$ cat recovery-phrase.txt | cardano-address key from-recovery-phrase Shelley
+xprv1zz8x2k6jemuq884k4s4c862n2maaskk07ua7xc4pcegfd70fx9ymnhrk5jkex5rh5fggph0682rdxjpaf97lkrmqqs49md3mq0n5ds4e07en8xnyn3g3f85zn85gapcwkht6y5djqrjmqdnqp2rg5nvmycwgyqvx
 ```
 
 Notice the `xprv` prefix to identify an e**x**tended **prv**ivate key. Should you prefer an hexadecimal output, you can choose a different output encoding via a special flag:
 
 ```
-$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley --base16
-48b95c9f19c1a174084615fd4a7a0d3ad2900f3902a59f2510df92f56c07a958
-de05de03906c441d7101963fe7cb9e9aa6c13dd833ee16156e2d675990a5fcb3
-10b63faf80237fb9ea51498250bc897cc7e19f7f97ae0352c44dbd8726c20f55
+$ cat recovery-phrase.txt | cardano-address key from-recovery-phrase Shelley --base16
+108e655b52cef8039eb6ac2b83e95356fbd85acff73be362a1c65096f9e93149
+b9dc76a4ad935077a25080ddfa3a86d3483d497dfb0f60042a5db63b03e746c2
+b97fb3339a649c51149e8299e88e870eb5d7a251b200e5b036600a868a4d9b26
 ```
 </details>
 
 <details>
-  <summary>How to generate a public stake key</summary>
+  <summary>How to generate an extended public stake key (public key plus chain code)</summary>
 
 ```
-$ cardano-address recovery-phrase generate --size 15 > recovery-phrase.prv
-$ cat recovery-phrase.prv \
-| cardano-address key from-recovery-phrase Shelley \
+$ cat recovery-phrase.txt | cardano-address key from-recovery-phrase Shelley \
 | cardano-address key child 1852H/1815H/0H/2/0 \
 | cardano-address key public
-xpub16y4vhpyuj2t84gh2qfe3ydng3wc37yqzxev6gce380fvvg47ye8um3dm3wn5a64gt7l0fh5j6sjlugy655aqemlvk6gmkuna46xwj9g4frwzw
+xpub16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7yak6lmcyst8yclpm3yalrspc7q2wy9f6683x6f9z4e3gclhs5snst6nr2z
+```
+
+> :information_source: `1852H/1815H/0H/2/0` is the derivation path that is typically used by Cardano wallet to identify a stake key within HD wallet. If you seek compatibility with Daedalus or Yoroi, use this as well!
+
+</details>
+
+<details>
+  <summary>How to generate a public stake key (public key without chain code)</summary>
+
+```
+$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley \
+| cardano-address key child 1852H/1815H/0H/2/0 \
+| cardano-address key public --pub
+xpub16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7qh83kg9
```

Comment:
or more precisely I would like `addr_vk` if we are dealing with payment/delegating address and `stake_vk` if we are dealing with stake address (reward account). If not possible we will need a way to have `--bech32WithHrp prefix`

Reply:
Perhaps use pub/xpub only in cardano-address, and show how to use the bech32 util to change HRP.

Reply:
What do you think of flipping this around and, have this being the default behavior, but require an explicit `--with-chain-code` flag to return the public key with chain code.
 
We should avoid returning the chain code unless the user knows what he/she is doing with it. Changing the default, we would also need to change a bit the error message when failing to parse bytes into a public key + chain code (so that it hints users in the right direction, that a 32-byte chain code is missing).

Reply:
I like that idea of requiring the user to specify what kind of key they need. Such as:
-  `(--with-chain-code | --without-chain-code )`.
- `cardano-address key xpub` /  `cardano-address key public`
- `( --extended-key | --key-only )`